### PR TITLE
add release notes for `edge-21.11.3`

### PR DESCRIPTION
**Pull request by Pothulapati**
This edge releases fixes a bug in the policy controller that was causing
compatibility issues with some Kuberentes distributions. This release also
includes a new High Availability mode for the gateway component in multicluster
extension. Various dependencies across the CNI, Policy Controller and dashboard
have also been upgraded. In the Proxy, Error logging when the proxy fails
to accept a connection due to a system error has been improved.

* Updated policy controller to use `openssl` instead of `rustls` to fix
  compatibility issues with some Kubernetes distros
* Added HA mode to multicluster gateway that adds a PodDisruptionBudget,
  additional replicas and anit-affinity to the deployment (thanks @Crevil)
* Improved TCP server error messages in the proxy
* Fixed broken Grafana links in the dashboard
* Upgraded CNI pkg to v0.8.1 in `linkerd-cni` to support latest CNI
  versions
* Updated various dependencies in the dashboard, policy controller
  (thanks @dependabot)

Signed-off-by: Tarun Pothulapati <tarunpothulapati@outlook.com>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-21.11.3
+
+This edge releases fixes a bug in the policy controller that was causing
```
Comment (olix0r) [inline CHANGES.md:5]: It's not really a bug in the policy controller... I might phrase this as "fixes a compatibility issue that prevented the policy controller from starting in some Kubernetes distributions"

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+compatibility issues with some Kuberentes distributions. This release also
+includes a new High Availability mode for the gateway component in multicluster
+extension. Various dependencies across the CNI, Policy Controller and dashboard
+have also been upgraded. In the Proxy, Error logging when the proxy fails
```
Comment (olix0r) [inline CHANGES.md:9]:
```suggestion
have also been upgraded. In the Proxy, error logging when the proxy fails
```

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-21.11.3
+
+This edge releases fixes a compatibility issue that prevented the policy controller
+from starting in some Kubernetes distributions. This release also includes a new
+High Availability mode for the gateway component in multicluster extension.
+Various dependencies across the CNI, Policy Controller and dashboard
+have also been upgraded. In the Proxy, error logging when the proxy fails
```
Comment (mateiidavid) [inline CHANGES.md:9]: Super nitpicky I know, sorry!

```suggestion
have also been upgraded. In the proxy, error logging when the proxy fails
```

File: CHANGES.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-21.11.3
+
+This edge releases fixes a compatibility issue that prevented the policy controller
+from starting in some Kubernetes distributions. This release also includes a new
+High Availability mode for the gateway component in multicluster extension.
+Various dependencies across the CNI, Policy Controller and dashboard
```
Comment (mateiidavid) [inline CHANGES.md:8]: TIOLI, would be explicit and say CNI plugin

```suggestion
Various dependencies across the CNI plugin, Policy Controller and dashboard
```

**Review by mateiidavid — APPROVED** [review]

**Review by olix0r — APPROVED** [review]

**Pull request merged**